spec/std/exception_spec.cr

```diff
@@ -0,0 +1,19 @@
+require "spec"
+
+class FooError < Exception
+  def message
```

Comment:
Due to how you use the specs I will suggest

```crystal
class FooError < Exception
  def message
    "#{super || ""} -- bar!"
  end
end
```

So that the spec tests that the base message implementation returns the value used at creation and that to_s / inspect_with_backtrace use the method rather than the ivar.

find a full sample at https://play.crystal-lang.org/#/r/17lx